### PR TITLE
refactor waitlist pages into single one

### LINKED ISSUE #33: [waitlist] refactor pages into single one

### PR DESCRIPTION
**Pull request by whyvrafvr**
fix #33

### CODE REVIEW

**Comment by whyvrafvr** [comment]
hey @s4m5ep10l, could you please review and approve ASAP ?

File: airdrops/SUMMARY.md [hunk 1]
```diff
@@ -1,8 +1,3 @@
 # Table of contents
 
 * [⛏ (WIP) Waitlist](README.md)
```
Comment (s4m5ep10l) [inline airdrops/SUMMARY.md:3]: @whyvrafvr  nice you merged all pages in a single one 👍

File: airdrops/README.md [hunk 2]
```diff
@@ -38,14 +38,104 @@ In the broader context of blockchain and decentralized platforms, where trustles
...
+
+***
 
 ## Future Improvements
+
+User data can be stored on blockchain, generating incentives for users and sponsors.
+
+Regarding the next **AirDrop**, /_activate_ **Response** may trigger an on-chain transaction, **saving user data in the blockchain** and **distributing free $POLN** **tokens** 💰 to users and sponsors.
+
+> Keep posted :tada:
```
Comment (s4m5ep10l) [inline airdrops/README.md:141]: We may use a hint...

**Review by s4m5ep10l — APPROVED** [review]

File: airdrops/README.md (hunk 2 above)
Comment (whyvrafvr) [reply airdrops/README.md:141]: Sure. Let's do it with an `info` hint

**Comment by s4m5ep10l** [comment]
Nice. I merge...

**Pull request merged**